support-frontend/assets/helpers/redux/commonState/reducer.ts

```diff
@@ -22,24 +21,11 @@ function getInternationalisationFromCountry(
 	};
 }
 
-function getABTestNames(abParticipations: Participations) {
-	const isUserInSupporterPlusABTest = 'supporterPlusOnly' in abParticipations;
-
-	const abTestType = isUserInSupporterPlusABTest
-		? { supporterPlusOnly: abParticipations.supporterPlusOnly }
-		: abParticipations;
-
-	return abTestType;
-}
-
 export const commonSlice = createSlice({
 	name: 'common',
 	initialState: initialCommonState,
 	reducers: {
 		setInitialCommonState(state, action: PayloadAction<CommonStateSetupData>) {
-			action.payload.abParticipations = getABTestNames(
-				action.payload.abParticipations,
-			);
 			const {
```

Comment:
Is this `action.payload.abParticipations =` logic not required to add the `abParticipations` to the common state?

Reply:
as far as I can tell this was just reassigning `action.payload.abParticipations` to itself, unless it contains `supporterPlusOnly`

Reply:
Ah I see, so were doing our `abParticipations` switcheroo in the Reducer, makes more sense to have that logic in `abtests.ts` as you've done for the new logic.